README.md

```diff
@@ -56,7 +85,7 @@ $ pulumi up
 
 ### Try it out with GHA
 
-[`.github/workflows/push-1password-secrets-to-gha`](https://github.com/conda-forge/infrastructure/blob/main/.github/workflows/push-1password-secrets-to-gha.yaml)
+[`.github/workflows/push-1password-secrets-to-heroku`](https://github.com/conda-forge/infrastructure/blob/main/.github/workflows/push-1password-secrets-to-heroku.yaml)
 
 Try it out by:
 * create and push a branch names with following the pattern "push-secrets-*" OR [manually run the workflow](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow)
```

Comment:
```suggestion
* create and push a branch named with the pattern "heroku-push-secrets-*" OR [manually run](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow) the [1Password-to-Heroku workflow](https://github.com/conda-forge/infrastructure/actions/workflows/push-1password-secrets-to-heroku.yaml)
```

Reply:
We should also update the bit for the 1Password to GHA section with the same format (link to the workflow to be run)